components/nvenergy.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "NV Energy Smart Thermostat",
+  "owner": ["@vegascom-jeff"],
+  "manifest": "https://raw.githubusercontent.com/vegascom-jeff/nvenergy/main/manifest.json",
```

Comment:
@vegascom-jeff I'm not a maintainer for this repo, just passing by - however, I've noticed that the manifest.json file has a syntax error (no comma after `issue_tracker` section) in the current revision (`f464722`).